index.html

```diff
@@ -123,24 +152,24 @@ <h2>About Myself</h2>
         <button>Get my resume</button>
         </div>
```

Comment:
- [ ] Kindly ensure that your **`About me`** element and **` Get my resume`** button matches the Figma design template chosen 👍🏽. You should add a line break to the  **`About me`** element and remove the line break from **`Get my resume`** button.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/186746030-7275bfe6-0037-4ca6-b32b-f2c2183c5e17.png)  | ![image](https://user-images.githubusercontent.com/63473584/186746117-0a58e84e-56a7-419f-8b73-1ea4ae674f97.png)  |

Reply:
done